### PR TITLE
Ensure security validations for post endpoints

### PR DESCRIPTION
**Pull request by froi**
# Why?
We have to ensure our post endpoints as secure from common threats and that our API isn't abused.

# What changed:

- Added [Helmet.js](https://helmetjs.github.io/)  - c5dcf1e
- Added rate limits with [express-rate-limit](https://www.npmjs.com/package/express-rate-limit) - 572d7d9

### CODE REVIEW

**Comment by froi** [comment]
[Ensure security validations around post endpoints #medium #ATO #Sprint_05 #Sprint_07](https://trello.com/c/tXopVbH3/121-ensure-security-validations-around-post-endpoints-medium-ato-sprint05-sprint07)

File: app.js [hunk 1]
```diff
@@ -31,13 +31,24 @@ const favicon             = require('serve-favicon');
...
+
 /* ------------------------------------------------------------------ *
                             API CONFIG
  * ------------------------------------------------------------------ */
 
 // define and configure express
 const app = express();
 const port = process.env.PORT || 3001;
+const limiter = new RateLimit({
+  windowMs: parseInt(process.env.WINDOW_MS, 10),
```
Comment (zachary-kuhn) [inline app.js]: Can we do defaults in case the environment variables aren't set?

File: app.js [hunk 2]
```diff
@@ -31,13 +31,24 @@ const favicon             = require('serve-favicon');
...
+  max: parseInt(process.env.MAX_IP_REQUESTS, 10),
+  delayMs:parseInt(process.env.DELAY_MS, 10),
+  headers: true
+});
+
+app.user(limiter);
```
Comment (zachary-kuhn) [inline app.js]: Is `user` a typo here?

File: app.js (hunk 1 above)
Comment (froi) [reply app.js]: Yep.

**Pull request merged**